modules/smarthubBidAdapter.js

```diff
@@ -61,31 +61,31 @@ function getPlacementReqData(bid) {
   };
 
   if (mediaTypes && mediaTypes[BANNER]) {
-    placement.adFormat = BANNER;
-    placement.sizes = mediaTypes[BANNER].sizes;
+    plcmt.adFormat = BANNER;
+    plcmt.sizes = mediaTypes[BANNER].sizes;
   } else if (mediaTypes && mediaTypes[VIDEO]) {
-    placement.adFormat = VIDEO;
-    placement.playerSize = mediaTypes[VIDEO].playerSize;
-    placement.minduration = mediaTypes[VIDEO].minduration;
-    placement.maxduration = mediaTypes[VIDEO].maxduration;
-    placement.mimes = mediaTypes[VIDEO].mimes;
-    placement.protocols = mediaTypes[VIDEO].protocols;
-    placement.startdelay = mediaTypes[VIDEO].startdelay;
-    placement.placement = mediaTypes[VIDEO].placement;
-    placement.skip = mediaTypes[VIDEO].skip;
-    placement.skipafter = mediaTypes[VIDEO].skipafter;
-    placement.minbitrate = mediaTypes[VIDEO].minbitrate;
-    placement.maxbitrate = mediaTypes[VIDEO].maxbitrate;
-    placement.delivery = mediaTypes[VIDEO].delivery;
-    placement.playbackmethod = mediaTypes[VIDEO].playbackmethod;
-    placement.api = mediaTypes[VIDEO].api;
-    placement.linearity = mediaTypes[VIDEO].linearity;
+    plcmt.adFormat = VIDEO;
+    plcmt.playerSize = mediaTypes[VIDEO].playerSize;
+    plcmt.minduration = mediaTypes[VIDEO].minduration;
+    plcmt.maxduration = mediaTypes[VIDEO].maxduration;
+    plcmt.mimes = mediaTypes[VIDEO].mimes;
+    plcmt.protocols = mediaTypes[VIDEO].protocols;
+    plcmt.startdelay = mediaTypes[VIDEO].startdelay;
+    plcmt.placement = mediaTypes[VIDEO].placement;
```

Comment:
This is the only line that matters here, you must also add placement or plcmt.plcmt

Reply:
resolved